### ISSUE TITLE
How to resize DataTable, according to different screen size?

### ISSUE THREAD

**Issue by Satyam1Vishwakarma** [flutter/flutter#110344]
I can not resize the DataTable, according to new screen size.
I am using [https://pub.dev/packages/sizer](url) but it doesnt help

**Comment by shubhamt619** [comment]
Hi @Satyam1Vishwakarma , Could you please share the code that you have tried ?
Thanks.

**Comment by iapicca** [comment]
> I am using [https://pub.dev/packages/sizer](url) but it doesnt help

I'd stay away from packages[ w/o real tests](https://github.com/TechnoUrmish/Sizer/blob/master/test/sizer_test.dart)

**Comment by shubhamt619** [comment]
Let me rephrase my request, Could you please share your source code where you are using datatable ?

**Comment by darshankawar** [comment]
@Satyam1Vishwakarma 
Your question is better suited for support channels like StackOverflow, so please ask the same there.
Closing from here. If you think there's a genuine bug in framework, open a new issue using template and fill out requested details so that we can address it properly.
